java/integTest/src/test/java/glide/cluster/CommandTests.java

```diff
@@ -1663,6 +1659,7 @@ public void fcall_readonly_function() {
         String code = generateLuaLibCode(libName, Map.of(funcName, "return 42"), false);
 
         assertEquals(libName, clusterClient.functionLoad(code, false).get());
+        Thread.sleep(1000); // let replica sync with the primary node
```

Comment:
Sleep is a recipe for flakiness. I am not sure how hard it is, but to completely remove the flakiness we should add a more reliable method like polling the log for a specific log line that indicate that the sync is completed

Reply:
Fixed